src/main/java/task/Task.java

```diff
@@ -0,0 +1,38 @@
+package task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[\u2713]" : "[\u2718]"); //return tick or X symbols
+    }
+    public String getTaskType(){
+        return "To be overridden by subclass";
+    }
+    public String getDescription() {
+        return description;
+    }
+    public boolean getDoneStatus(){
+        return isDone;
+    }
+
+    public void markAsDone() {
```

Comment:
Try to standardize your getters and setters. If they are using the same variable "isDone", then the getter can be _get_**DoneStatus**(), setter can be _set_**DoneStatus**(), so that you know you are modifying the same "isDone" variable.